etl/steps/data/garden/aviation_safety_network/2023-04-18/aviation_statistics.meta.yml

```diff
@@ -0,0 +1,68 @@
+dataset:
+  title: Fatal aviation accidents and fatalities (ASN, 2023-04-18)
+  description: |
+    Data on fatal accidents, hijacking incidents and numbers of fatalities are based on airliners of 14+ passengers, and do not include corporate jet and military transport accidents.
+
+    Air traffic figures are based on the number of airliner flight departures (not the number of passengers) per year.
+  sources:
+    -
+      name: Aviation Safety Network
```

Comment:
This is the name that is surfaced in charts, right?

I tend to have the publication year, too, as that is our standard on charts. So I'd have `Aviation Safety Network (2023)` instead.

In the future, I think charts should bake the source name based on fields `name` and `publication_year` to produce something like `[name] ([publication_year])`. So in that case, the year wouldn't be required in the field `name`.